PDF Add and reformat for v11 (#794)
vitessio/website: content/en/docs/design-docs/query-serving/reserved-conn.md

Once a session has been marked for reserved connections, it will stay as such until the user disconnects.

### Temporary tables and reserved connections

Temporary tables exist only in the context of a particular MySQL connection.
If a user uses temporary tables, Vitess will mark the session as needing a reserved connection. It will continue to require a reserved connection until the user disconnects - removing the temp table is not enough. More info can be found [here.](/docs/reference/compatibility/mysql-compatibility/#temporary-tables)

### GET_LOCK() and reserved connections

> The MySQL locking functions allows users to work with user level locks. Since the locks are tied to the connection, and freeing lock has to be done in the same connection as the lock was acquired, use of these functions will force a connection to become a reserved connection. This connection is also kept alive so it does not time out due to inactivity. More information can be found [here.](/docs/design-docs/query-serving/locking-functions/).

Reviewer: ```suggestion
The MySQL locking functions allow users to work with user level locks. Since the locks are tied to the connection, and the lock must be released in the same connection as it was acquired, use of these functions will force a connection to become a reserved connection. This connection is also kept alive so it does not time out due to inactivity. More information can be found [here.](/docs/design-docs/query-serving/locking-functions/).
```

Author: Changes made. Thank you!!!
